docs/faq/data-validation.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "12efa51a-6173-473d-b44e-20056d1bf2f0",
+   "metadata": {},
+   "source": [
+    "# Data Validation with the LaminDB ORM"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "28214be7-db6e-4a96-b354-74520470980e",
+   "metadata": {},
+   "source": [
+    "LaminDB implements data validation at the ORM level by fully integrating the SQLModel ORM with pydantic type checking.\n",
+    "\n",
+    "Let's take a look at data validation behavior in LaminDB."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5759ee45-c50d-4e30-bbd0-e25e254ff758",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lamindb as ln\n",
+    "import lamindb.schema as lns\n",
+    "import pytest\n",
+    "from pydantic import ValidationError"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df946dfd-fe74-4705-bc1d-a600b41cc15a",
+   "metadata": {},
+   "source": [
+    "## Missing Field Error"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "505527e7-d0b6-4548-a9b4-7a91e5eba908",
+   "metadata": {},
+   "source": [
+    "Let's create a `User` instance without the required `email` and `handle` fields."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "63125633-b71e-4b33-8001-8b752c93b144",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "??lns.User"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a736c303-0bde-4062-9159-2d61dad8f321",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with pytest.raises(ValidationError) as e:\n",
+    "    exception = e\n",
+    "    user_missing = lns.User(id=\"123\")\n",
+    "print(exception.exconly())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "716dd427-e10c-4fad-bb4a-086633747a35",
+   "metadata": {},
+   "source": [
+    "## Simple Type Error"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "229b7247-1719-4e72-aac4-976002371f44",
+   "metadata": {},
+   "source": [
+    "Let's create a `Pipeline` instance with the wrong type for the optional field `name`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6d385d2-08ed-438f-a61c-56ff7f12ab0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "??lns.Pipeline"
```

Comment:
This here prints nothing on the generated docs:
<img width="723" alt="image" src="https://user-images.githubusercontent.com/16916678/214051645-ef0dff3d-5a88-4460-9b5a-46baba0c6f8b.png">

Reply:
Removed it!